### ISSUE TITLE
Design Intent - Accessibility Feedback - Transition Experience, tbd, V0.1 Transition Experience

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#81749]
## Next Steps for the VFS team

- [ ] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [ ] **Close the ticket** when all feedback has been addressed.

## Thoughts/questions

- Thank you for answering all of our questions! I'm looking forward to seeing this one come back for Midpoint! 

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- [ ] **Must:** As you refine your mockups, make sure you follow material honesty principles.
   - If it's navigational, it's coded as a link and visually looks like a link
   - If it performs an action on the page (changes to content, submits user data, opens/closes modals, etc), it's coded as a button and visually looks like a button
   - If it's a form input for collecting user responses (eg. checkbox, radio button, text input), it's coded with the appropriate HTML element and is visually recognizable

   And as you hand off to your engineering team -- whenever possible, use existing design system components. When that's not possible, use existing semantic HTML (not `div` or `span` elements with a bunch of JavaScript to make them interactive).
   
   Various assistive technologies have different interaction patterns depending on how the element is coded, so we need to make sure each element is doing what it looks like it's doing and has all of the appropriate accessibility functionality built in.
- [ ] **Must:** You mentioned the importance of the "Copy URL to save or share" option for your results. Looking at that feature on the REACH site, the interaction there doesn't follow the material honesty principles and doesn't announce to screen reader users that anything has happened when you select the link. I definitely agree with the value of the feature! But I'd recommend you do some design exploration to see if you can improve on what was done on REACH.
- [ ] **Must:** Probably something you're already thinking about, but not currently in the mockup: Make sure you provide users with some way to go back and correct their responses, if they realize they've made a mistake.
- [ ] **Engineering note:** I suspect there may be some additional context that will be needed for some of these screens (example: "feel like you're struggling" may need to be qualified to differentiate between the kind of help that REACH provides and a general "yeah I'm frustrated with VA"). Whenever that kind of explanatory text is needed, make sure it's programmatically associated with the interactive elements to ensure that screen reader users get the appropriate context.

## Governance team actions
- [ ] Format feedback as individual tasks (check boxes)
- [ ] Assign this ticket to the VFS team member that opened the Slack request
- [ ] Add the VFS team product label
- [ ] Add the VFS team the feature label (if applicable)
- [ ] Add the touchpoint labels
- [ ] Add the practice area labels
- [ ] Add the Collaboration Cycle initiative milestone
- [ ] Add the Collaboration Cycle initiative epic
